src/openforms/registrations/tasks.py

```diff
@@ -74,7 +89,22 @@ def register_submission(submission_id: int) -> Optional[dict]:
 
     submission.registration_status = status
     submission.registration_result = result_data
-    submission.save(update_fields=["registration_status", "registration_result"])
+    submission.save(
+        update_fields=[
+            "registration_status",
+            "registration_result",
+        ]
+    )
+
+
+@app.task
+def resend_submissions():
+    for submission in Submission.objects.filter(
+        registration_status=RegistrationStatuses.failed,
+        completed_on__gte=timezone.now()
+        - timedelta(hours=settings.CELERY_BEAT_RESEND_SUBMISSIONS_TIME_LIMIT),
```

Comment:
you can move this calculation out of the loop, which will in turn make the code more readable